Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -96,8 +96,9 @@ extension HomeViewController: UICollectionViewDelegate, UICollectionViewDelegate
             let checkInListHeight = checkInListItemCellHeight * habits.count
             let bottomSectionInset = 32
             return CGSize(width: width, height: CGFloat(checkInListHeight + bottomSectionInset))
-        case .allGyms:
-            return CGSize(width: width, height: 123)
+        case .myGyms:
+            let height: CGFloat = (GymsListCell.itemHeight + GymsListCell.minimumLineSpacing) * ceil(CGFloat(integerLiteral: myGyms.count) / 2) + GymsListCell.sectionInsetBottom - GymsListCell.minimumLineSpacing //Height is calculated as item height of every row of 2 cells + minimum line spacing between them + bottom section inset value
```

Comment:
lets put the comment on the line before & also split this line into multiple ones so that its easier to read.